wordpress-mobile/WordPress-Android
# Facebook Publicize disable selection of profiles

Fixes #7591 & #8026 

We're no longer allowing users to add their profiles to publicize. Also, this adds pages to the selection list where people can connect to.

To test:

1. Add a facebook account with a page. Only the page will be shown in the selection dialog. No profiles.
2. Add a facebook account with no pages. It will error out.
3. See the facebook detail item and notice the new user message about the changes August 1st. 

![screenshot_20180729-195923](https://user-images.githubusercontent.com/1158819/43375451-fdead33e-9369-11e8-840e-9adf46776841.png)

@loremattei we're going to make a 10.5.1 to release this by August 1. Feel free to ship 10.5 without this tomorrow, thanks!
